doc/installation_linux_manual.md

```diff
@@ -0,0 +1,169 @@
+    </li>
+    <li>
+        <a href="#building-librealsense2-sdk">Building librealsense2 SDK</a>
+    </li>
+    <li>
+        <a href="#troubleshooting-installation-and-patch-related-issues">Troubleshooting Installation and Patch-related Issues</a>
+    </li>
+  </ol>
+</details>
+
+**Note:** Due to the USB 3.0 translation layer between native hardware and virtual machine, the *librealsense* team does not support installation in a VM. <br/>
+If you do choose to try it, we recommend using VMware Workstation Player, and not Oracle VirtualBox for proper emulation of the USB3 controller.
+<br/><br/> Please ensure to work with the supported Kernel versions listed [here](https://github.com/IntelRealSense/librealsense/releases/) and verify that the kernel is updated properly according to the instructions.
+
+## Prerequisites
+
+**Note:** The scripts and commands below invoke `wget, git, add-apt-repository` which may be blocked by router settings or a firewall. <br/>
+Infrequently, apt-get mirrors or repositories may also cause timeout. For _librealsense_ users behind an enterprise firewall, <br/>
+configuring the system-wide Ubuntu proxy generally resolves most timeout issues.
+
+**Important:** Running RealSense Depth Cameras on Linux requires patching and inserting modified kernel drivers. <br/>
+Some OEM/Vendors choose to lock the kernel for modifications. Unlocking this capability may require modification of BIOS settings
+
+## Install Dependencies
+
+1. Make Ubuntu up-to-date including the latest stable kernel:
+   ```sh
+   sudo apt-get update && sudo apt-get upgrade && sudo apt-get dist-upgrade
+   ```
+2. Install the core packages required to build _librealsense_ binaries and the affected kernel modules:
+   ```sh
+   sudo apt-get install libssl-dev libusb-1.0-0-dev libudev-dev pkg-config libgtk-3-dev cmake
+   ```
+   **Cmake Note:** certain _librealsense_ [CMAKE](https://cmake.org/download/) flags (e.g. CUDA) require version 3.8+ which is currently not made available via apt manager for Ubuntu LTS.
+3. Install tools
+   ```sh
+   sudo apt-get install git wget cmake
+   ```
+4. Prepare Linux Backend and the Dev. Environment
+   <br/>
+   Unplug any connected Intel RealSense camera and run:
+   <br/>
+   ```sh
+   sudo apt-get install libglfw3-dev libgl1-mesa-dev libglu1-mesa-dev at
+   ```
+5. Install IDE (Optional):
+   <br/>
+   We use [QtCreator](https://wiki.qt.io/Install_Qt_5_on_Ubuntu) as an IDE for Linux development on Ubuntu.
+
+## Install librealsense2
+
+1. Clone/Download the latest stable version of _librealsense2_ in one of the following ways:
+   * Clone the _librealsense2_ repo
+     ```sh
+     git clone https://github.com/IntelRealSense/librealsense.git
+     ```
+   * Download and unzip the latest stable _librealsense2_ version from `master` branch <br/>
+     [IntelRealSense.zip](https://github.com/IntelRealSense/librealsense/archive/master.zip)
+
+   **Note**: <br/>
```

Comment:
Lets move this us to the end of "Install Dependencies"

Reply:
Done